### PR TITLE
Added language starting with v to dropdown file

### PR DESCRIPTION
**Pull request by shubhamburad**

### CODE REVIEW

File: A-Z-dropdown.html [hunk 1]
```diff
@@ -762,6 +762,7 @@
         <option value="visual lisp">Visual LISP</option>
         <option value="visual objects">Visual Objects</option>
         <option value="visual prolog">Visual Prolog</option>
+        <option value="Viper (Ethereum/Ether (ETH))">Visual Prolog</option>
```
Comment (chilarai) [inline A-Z-dropdown.html:765]: I can still see Visual Prolog in the name. Also, viper should have been Vyper. Isn't it?

**Review by chilarai — CHANGES_REQUESTED** [review]
Please do yhe required changes

File: A-Z-dropdown.html (hunk 1 above)
Comment (shubhamburad) [reply A-Z-dropdown.html:765]: Officialy it should be viper so can i raise pr again and can you please approve it?

Comment (chilarai) [reply A-Z-dropdown.html:765]: @Shrutiswift has already added that in PR #132. Thanks

**Pull request closed without merging**